specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -1444,6 +1875,26 @@
         },
         "cost": {
           "$ref": "#/definitions/PhoneNumberCost"
+        },
+        "operatorId": {
+          "description": "Id of the operator that provided the number",
+          "type": "string"
+        },
+        "operatorName": {
+          "description": "Name of the operator that provided the number",
+          "type": "string"
+        },
+        "phoneNumberSource": {
+          "description": "Source of the number, e.g. Cloud or OperatorConnect",
+          "enum": [
+            "cloud",
+            "operatorConnect"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberSource",
+            "modelAsString": true
+          }
```

Comment:
These properties are added from an existing preview API `2022-06-01-preview`: https://github.com/Azure/azure-rest-api-specs/tree/main/specification/communication/data-plane/PhoneNumbers/preview/2022-06-01-preview

They appear in the diff because they weren't present in the preview API this is based upon, `2023-05-01-preview`.  The only change made is that they weren't marked as required to avoid breaking changes; which should be fine because they are actually readonly (I didn't add this to the properties because it wasn't in the original spec).